lib/process-services-cloud/src/lib/process/process-filters/components/process-filters-cloud.component.ts

```diff
@@ -220,4 +236,47 @@ export class ProcessFiltersCloudComponent implements OnInit, OnChanges, OnDestro
     isActiveFilter(filter: ProcessFilterCloudModel): boolean {
         return this.currentFilter.name === filter.name;
     }
+
+    initProcessNotification(): void {
+        if (this.appName && this.enableNotifications) {
+            this.processFilterCloudService
+                .getProcessNotificationSubscription(this.appName)
+                .pipe(debounceTime(1000), takeUntil(this.onDestroy$))
+                .subscribe((result: TaskCloudEngineEvent[]) => {
+                    this.updateFilterCounters();
+                });
+        }
+    }
+
+    updateFilterCounters(): void {
+        this.filters.forEach((filter: ProcessFilterCloudModel) => {
+            if (filter?.status) {
+                this.updateFilterCounter(filter);
+            }
+        });
+    }
+
+    updateFilterCounter(filter: ProcessFilterCloudModel): void {
+        this.counters$[filter.key] = this.processListCloudService.getProcessCounter(filter.appName, filter.status).pipe(
+            tap((filterCounter) => {
+                this.checkIfFilterValuesHasBeenUpdated(filter.key, filterCounter);
+            })
+        );
+    }
+
+    checkIfFilterValuesHasBeenUpdated(filterKey: string, filterValue: number): void {
+        if (!this.currentFiltersValues[filterKey]) {
```

Comment:
This condition can be sorted to avoid the need of setting this empty return here.

Reply:
What about something like `this.currentFiltersValues[filterKey] && this.currentFiltersValues[filterKey] !== filterValue) { do something } else { assign the filter}` or if we don't really care about if the filter it's already there or no something like `checkIfFilterValuesHasBeenUpdated(filterKey: string, filterValue: number): void {
    if (this.currentFiltersValues[filterKey] !== filterValue) {
        this.currentFiltersValues[filterKey] = filterValue;
        if (this.currentFiltersValues[filterKey] !== undefined) {
            this.updatedFilter.emit(filterKey);
            this.updatedFiltersSet.add(filterKey);
        }
    }
}`